frontend/src/pages/article/article.tsx

```diff
@@ -27,16 +46,16 @@ const Article: FC = () => {
     { id: 5, name: 'Tech' },
   ];
 
-  const MOCKED_ARTICLE = {
-    title: 'Modern Full-Stack Developer Tech Stack 2021',
-    text: MOCKED_TEXT,
+  const ARTICLE = {
+    title,
+    text,
     tags: MOCKED_TAGS,
   };
 
   return (
     <Layout>
       <div className={styles.container}>
-        <ArticleView article={MOCKED_ARTICLE} />
+        <ArticleView article={ARTICLE} />
```

Comment:
To pass mocked tags for now (btw what is currently passed as mocked tags - should be a prompt data (if article was generated with prompt)
```suggestion
        <ArticleView article={{ title, text, tags: MOCKED_TAGS }} />
```